app/src/main/kotlin/com/intive/picover/profile/view/ProfileScreen.kt

```diff
@@ -36,3 +69,55 @@ fun ProfileScreen(viewModel: ProfileViewModel) {
 		}
 	}
 }
+
+@Composable
+private fun LabelText(
```

Comment:
I don't think that this needs wrapping as we have only usage of composable Text here, that could be just declared in the main layout code part

Reply:
Done 😄